management/http.go

```diff
@@ -28,6 +28,21 @@ func (client client) SendAzurePostRequest(url string, data []byte) (OperationID,
 	return client.doAzureOperation("POST", url, "", data)
 }
 
+func (client client) SendAzurePostRequestWithReturnedResponse(url string, data []byte) (OperationID, []byte, error) {
+	resp, err := client.sendAzureRequest("POST", url, "", data)
+	if err != nil {
+		return "", nil, err
+	}
+
+	opID, err := getOperationID(resp)
```

Comment:
hmm if you're going to create a new method that's only going to be used by SQL Azure, no need for returning `OperationID`, because operation is not asynchronous and we return the operation ID in the error if that happens to be the case.